plugin.go

```diff
@@ -87,9 +87,16 @@ func (c *Plugin) Run(cliConnection plugin.CliConnection, args []string) {
 			return config.Encrypt(cliConnection, configServerInstanceName, plainText, fileToEncrypt, authClient)
 		})
 
+	case "config-server-add-git":
```

Comment:
Also need to register the `config-server-add-git` command in the `GetMetadata` function near the bottom of the file.